templates/maas/_blog.html

```diff
@@ -0,0 +1,42 @@
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <h2 class="p-muted-heading">Latest from our blog</h2>
```

Comment:
Another design related comment: there's a bit more space below this heading in the design.